### PR TITLE
Applies trails to torque/heatmap css when selected in editor

### PR DESCRIPTION
**Pull request by fdansv**
@javisantana

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/style/style-converter.js [hunk 1]
```diff
@@ -336,6 +336,19 @@ function generateCartoCSS (style, geometryType) {
     css += renderBlock({ labels: styleDef.labels }, geometryType);
     css += '}';
   }
+
+  if (styleDef.animated && styleDef.animated.trails && styleDef.animated.trails > 0) {
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor/style/style-converter.js]: better use animatedEnabled(styleDef) && styleDef.animated.trails && styleDef.animated.trails > 0

File: lib/assets/javascripts/cartodb3/editor/style/style-converter.js [hunk 2]
```diff
@@ -336,6 +336,19 @@ function generateCartoCSS (style, geometryType) {
...
+    css += '\n' +
+    _.range(1, parseInt(styleDef.animated.trails, 10) + 1)
+      .map(function (t) {
+        return '#layer[frame-offset=' + t + '] {\n  marker-width: ' +
+          (styleDef.fill.size.fixed + 2 * t) +
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor/style/style-converter.js]: what happens when a ramp is enabled? fixed in that case could have no value

File: lib/assets/javascripts/cartodb3/editor/style/style-converter.js [hunk 3]
```diff
@@ -336,6 +336,19 @@ function generateCartoCSS (style, geometryType) {
...
+          ';\n  marker-fill-opacity: ' +
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor/style/style-converter.js]: use renderBlock to generate css

**Comment by javisantana** [comment]
could you add some tests?

**Comment by fdansv** [comment]
@javisantana tests adapted to include trails, followed indications. OK?

File: lib/assets/javascripts/cartodb3/editor/style/style-converter.js [hunk 4]
```diff
@@ -336,9 +342,29 @@ function generateCartoCSS (style, geometryType) {
...
 }
 
+function trails (def) {
+  var baseWidth = def.fill.size && parseInt(def.fill.size.fixed, 10);
+  var baseOpacity = parseInt(def.fill.color.opacity, 10) || 1;
+  if (!baseWidth) return '';
+  return '\n' +
+    _.range(1, parseInt(def.animated.trails, 10) + 1)
+      .map(function (t) {
+        return '#layer[frame-offset=' + t + '] {\n  marker-width: ' +
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor/style/style-converter.js]: you should be using makeCartoCSS and not generating cartocss for properties on the fly (it's fine for the condition)

**Pull request merged**